rllib/core/learner/tests/test_learner_group.py

```diff
@@ -19,7 +25,7 @@
 
 REMOTE_SCALING_CONFIGS = {
     "remote-cpu": LearnerGroupScalingConfig(num_workers=1),
-    "remote-gpu": LearnerGroupScalingConfig(num_workers=1, num_gpus_per_worker=0.5),
+    "remote-gpu": LearnerGroupScalingConfig(num_workers=1, num_gpus_per_worker=1),
```

Comment:
Why did we change this? Would it break if we used fractional GPUs here?

Reply:
this learner/group actually won't even take fractional gpus. so it is pointless. I changed it while I was doing some debugging